Configure

```diff
@@ -302,12 +302,9 @@ $config{openssldir}="";
 $config{processor}="";
 $config{libdir}="";
 $config{cross_compile_prefix}="";
-$config{fipslibdir}="/usr/local/ssl/fips-2.0/lib/";
-my $nofipscanistercheck=0;
 $config{baseaddr}="0xFB00000";
```

Comment:
baseaddr thing was originally introduced to make fips module work [on 32-bit Windows]. Now with new build system it's not used at all. In other words it can be removed too...